test_scripts/Resumption/InteriorVehicleData/001_InteriorVD_resumption_one_moduleType_without_moduleId_unexpected_disconnect.lua

```diff
@@ -28,8 +28,8 @@ local appSessionId = 1
 
 --[[ Local Functions ]]
 local function checkResumptionData()
-  local defaultModuleId = 1
-  common.checkModuleResumptionData(moduleType, common.getModuleId(moduleType, defaultModuleId))
+  local defaultModuleNumber = 1
```

Comment:
Please also update canstant name `defaultModuleId` to `defaultModuleNumber` in 011, 012, 021, 015

Reply:
Done (see 66560cb )